src/core/dev/ring_slave.cpp

```diff
@@ -913,9 +917,14 @@ bool steering_handler<KEY4T, KEY2T, HDR>::rx_process_buffer_no_flow_id(
             sz_data, ip_tot_len);
         ring_loginfo("Rx packet info (buf->%p, bufsize=%zu), id=%s", p_rx_wc_buf_desc->p_buffer,
                      p_rx_wc_buf_desc->sz_data, hdr_get_id(p_ip_h).c_str());
-        vlog_print_buffer(VLOG_INFO, "rx packet data: ", "\n",
-                          (const char *)p_rx_wc_buf_desc->p_buffer,
-                          std::min(112, (int)p_rx_wc_buf_desc->sz_data));
+
```

Comment:
can you explain the change? thanks

Reply:
sure - the old implementation has printed at most 112 bytes from rx here.

The implementation was complex and used old logger implementation details.
it also was used only once which imho didn't justify extracting to a function in a different cpp file.

I allowed myself to remove it - I got those bytes in a c++ way and gave it to the logger.

due to your comment - I introduce a static function here with a readable name